src/FileUploader.ts

```diff
@@ -66,6 +66,7 @@ export class FileUploader {
     return this.requestClient
       .post(uploadUrl, formData, undefined, 'application/json', headers)
       .then(async (res) => {
+        this.requestClient!.appendRequest(res, sasJob, this.sasjsConfig.debug)
```

Comment:
I think it'd be best if we didn't have to ask the request client to append the request - it should be able to do it internally whenever get/post/delete/patch/put are called.

Reply:
As I understand it, there are a few issues with this suggestion:
* Which request to attach it to (some SAS calls comprise several network requests)
* How to capture the necessary metadata (program path, debug state, log, response, etc)